### PR TITLE
update interface pod

### PR DESCRIPTION
**Pull request by gao-feng**
work together with hyperstart patch `https://github.com/hyperhq/hyperstart/pull/2`

```
{
"interfaces": [{
    "bridge": "qbr-aaaa",
    "Ifname": "tap-aaaa",
    "ip": "10.10.0.8/24",
    "gw":"10.10.0.1"
},
{
    "bridge": "qbr-bbbb",
    "mac": "52:54:c3:14:59:46",
    "ip": "192.168.2.2/24"
}],
"dns": ["114.114.114.114", "8.8.8.8"]
}

### CODE REVIEW

**Comment by carmark** [comment]
I prefer the second propose.

File: hypervisor/network/network_linux.go [hunk 1]
```diff
@@ -1013,11 +1014,115 @@ func Allocate(vmId, requestedIP string, index int, addrOnly bool, maps []pod.Use
...
+			Bridge:      config.Bridge,
+			IPPrefixLen: maskSize,
+			Device:      "",
+			File:        nil,
+			Automatic:   false,
+		}, nil
+	}
+
+	tapFile := new(os.File)
+	tapFile, err = os.OpenFile("/dev/net/tun", os.O_RDWR, 0)
```
Comment (laijs) [inline hypervisor/network/network_linux.go]: the `tapFile := new(os.File)` is unneeded
`tapFile, err := os.OpenFile("/dev/net/tun", os.O_RDWR, 0)` is enough. (note the `:=`)

**Comment by laijs** [comment]
It seems good

**Comment by gao-feng** [comment]
@laijs updated, please review, thanks!

**Pull request merged**